science/2022-11-23-science-blog-6/index.md

```diff
@@ -1,15 +1,261 @@
\ No newline at end of file
+1. [The most important findings in a nutshell: numbers, data and facts](#1-the-most-important-findings-in-a-nutshell-numbers-data-and-facts)
+2. [Positivity rates (nationwide)](#2-positivity-rates-nationwide)
+    1. [RAT](#21-rat)
+    2. [PCR](#22-pcr)
+3. [Local differences in incidence](#3-local-differences-in-incidence)
+4. [Positivity rates (local))](#4-positivity-rates-local)
+    1. [RAT](#41-rat)
+    2. [PCR](#42-pcr)
+5. [Positivity rate and incidence](#5-positivity-rate-and-incidence)
+    1. [RAT](#51-rat)
+    2. [PCR](#52-pcr)
+6. [Differences between urban and rural districts](#6-differences-between-urban-and-rural-districts)
+7. [Conclusion](#7-conclusion)
+8. [Continued further](#8-continued-further)
+
+## 1 The most important findings in a nutshell: numbers, data and facts
+
+- Whether rapid antigen test or PCR test, the positivity rate of red warnings is always greater than that of green warnings (low risk with risk exposure). In other words, where risk exposures are detected, the CWA can reliably assess the risk.
+
+- For rapid antigen tests, the positivity rate (the proportion of all tests performed that are positive) after a red warning is at all times greater than the positivity rates after a green warning (lower risk with risk exposure) or low risk without risk exposure.
+
+- For PCR tests, the positivity rate after a red warning is always greater than the positivity rates after a green warning (lower risk with risk exposure). In times of high incidence, the positivity rate after a red warning is in part smaller than at lower risk without risk exposure - here a strong preselection by positive antigen rapid tests is to be assumed.
+
+- The increased positivity rate after a red warning compared to the positivity rates at lower risk with or without a risk exposure cannot be solely explained by local differences in incidence.
+
+- For rapid antigen tests and PCR tests, the dependence of a positive test result on incidence after a red warning is only about half as high as for a lower risk without a risk exposure. 
+
+- The differences between the positivity rates and thus the significance of a red warning with regard to the risk of infection are clearer in rural areas than in urban ones.
+
+
+## 2 Positivity rates (nationwide)
+
+So far, approx. 36.8 million test results have been transmitted via the CWA data donation (as of: 24. October 2022). Only test results that were positive or negative were taken into account (no pending and invalid ones). Risk exposures identified by the CWA were only taken into account if they were up to 10 days prior to test registration (this was the case for approximately 10.5 million test results). An indication of the districts (administrative district or urban district of the CWA user's place of residence) was available for approx. 14.1 million test results.
```

Comment:
"no pending and invalid ones" or
"no pending or invalid ones" ?

Reply:
"no pending or invalid ones"

Reply:
Thanks for your suggestion, it has been implemented in https://github.com/corona-warn-app/cwa-website/pull/3363/commits/4c8e51aead1712b20d50c96a4cd72ac3b0e3bcb5